ConnectSDKTests/Services/Helpers/WebOSTVServiceSocketClientTests.m

```diff
@@ -95,4 +95,58 @@ - (void)testDeniedPairingShouldCallRegistrationFailed {
                                  }];
 }
 
+-(void)testPairingShouldCallSocketWillregister{
+    id serviceMock = OCMClassMock([WebOSTVService class]);
+    id webSocketMock = OCMClassMock([LGSRWebSocket class]);
+    //Send Pairing type which is not supported by the TV. Supported pairing type is PROMPT
+    OCMStub([serviceMock pairingType]).andReturn(DeviceServicePairingTypeMixed);
+    id socketClientDelegateMock = OCMProtocolMock(@protocol(WebOSTVServiceSocketClientDelegate));
+    OCMStub([socketClientDelegateMock socket:OCMOCK_ANY didReceiveMessage:OCMOCK_ANY]).andReturn(YES);
+    XCTestExpectation *socketWillRegisterCalled = [self expectationWithDescription:@"socketWillRegisterCalled: is called"];
```

Comment:
You probably meant `@"socketWillRegister: is called"` as the expectation string.